two_points_on_the_same_branch.sh

```diff
@@ -248,6 +248,9 @@ for coq_opam_package in $coq_opam_packages; do
         opam list
         echo "DEBUG: coq_opam_package = $coq_opam_package"
         opam show $coq_opam_package || continue 2
+    else
+        # cause to skip with error if unknown package
+        opam show $coq_opam_package >/dev/null || continue 2
```

Comment:
Maybe you'd like to add some message error "Unknown package foo"?

Why is CI failing? [Coq bignums fail to intall]

I am not expert on bash, so we could remove -e if you want, tho I am more comfortable handling errors explicitly.

Reply:
>Maybe you'd like to add some message error "Unknown package foo"?

opam will say that on stderr

Reply:
>Why is CI failing? [Coq bignums fail to intall]

No clue, we should move to gitlab so we can upload logs as artifacts.

Reply:
I'm all for it; but I won't have time to do that soon, maybe we could build with `-v` if `BENCH_DEBUG` is set?